src/feedlist_formaction.cpp

```diff
@@ -82,7 +83,8 @@ void feedlist_formaction::prepare()
 	std::string sort_order =
 		v->get_cfg()->get_configvalue("feed-sort-order");
 	if (sort_order != old_sort_order) {
-		v->get_ctrl()->sort_feeds();
+		v->get_ctrl()->get_feedhandler()->sort_feeds(
```

Comment:
Because I liked the idea that FeedHandler doesn't have to be aware that it has to tokenize the input. Yeah, I know that it still knows that first item of vector is sortmethod and the next is direction... so it's not really a strong argument. 

Ideally, I would like to pass it the sortmethod and direction (and more globally, I don't like that methods are aware how they should handle the input [handle_action(const string& action), anyone? :)] but that's not the discussion for now. :D)

The more I think about it, the more I want to revert this.

Reply:
> Ideally, I would like to pass it the sortmethod and direction (by enum classes?).

A `stuct` with two fields, both `enum class`es, I guess. I agree, but it's outside of scope for this PR. We can do this later when we have the current implementation under tests.

`handle_action` is this way by necessity, since it deals with user config, which is untyped. But I agree that the rest of the methods should accept something more structured than a string (if possible).